trunk/SUAVE/Methods/Aerodynamics/AVL/Data/Body.py

```diff
@@ -63,3 +63,4 @@ def append_section(self,section,orientation='Horizontal'):
 			raise KeyError('No key, "{}". Use "Horizontal" or "Vertical".'.format(orientation))
 		return
 
+  
```

Comment:
I clearly didn't add a random blank line here. The commit history has me changing the exception type a year and a half ago.

Remove the extra blank line at the end.